src/models.ts

```diff
@@ -0,0 +1,31 @@
+export interface JsonRpcRequest {
+  jsonrpc: string;
+  id: string;
+  method: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  params: Array<any>;
+}
+
+export interface RpcBundle {
+  txs: Array<string>;
+  blockNumber: string;
+}
+
+export interface DuneBundleTransaction {
+  nonce: number;
+  maxFeePerGas: string | undefined;
+  maxPriorityFeePerGas: string | undefined;
+  gasPrice: string | undefined;
```

Comment:
is this equivalent to "Optional" or nullable? If so, I believe we could also use

```suggestion
  maxFeePerGas?: string;
  maxPriorityFeePerGas?: string;
  gasPrice?: string;
```